modules/optimization/test/tests/executioners/constrained/equality/quadratic_minimize_constrained.i

```diff
@@ -0,0 +1,29 @@
+[Optimization]
+[]
+
+[OptimizationReporter]
+  type = QuadraticMinimizeConstrained
+  parameter_names = 'results'
+  num_values = 3
+  initial_condition = '1 1 1'
+  objective = 0.0
+  solution = '1 2 3'
+  measurement_points = ''
```

Comment:
A parameter here for the solution_sum_equality constraint setting it to 5 will make it clear why the gold values add up to 5 instead of 6.